rskj-core/src/test/java/co/rsk/peg/utils/BridgeEventLoggerLegacyImplTest.java

```diff
@@ -306,7 +306,8 @@ public void testLogUnrefundablePegin() {
     @Test(expected = DeprecatedMethodCallException.class)
     public void testLogReleaseBtcRequestReceived() {
         String sender = "0x00000000000000000000000000000000000000";
-        byte[] btcDestinationAddress = "1234".getBytes();
+        String base58Address = "mipcBbFg9gMiCh81Kj8tqqdgoZub1ZJRfn";
+        Address btcDestinationAddress = Address.fromBase58(NetworkParameters.fromID(NetworkParameters.ID_REGTEST), base58Address);
```

Comment:
Just a proposal, you can simply use PegTestUtils. There is a method that will get a random btc address